worklets/Overview.bs

```diff
@@ -249,14 +251,19 @@ When a user agent is to <dfn export>create a WorkletGlobalScope</dfn>, given |wo
             |moduleResponsesMap|, |outsideSettings|, and |insideSettings| when
             it asynchronously completes.
 
-        3. <a>Run a module script</a> given |script|.
+        3. Let |promise| be the result of <a lt="run a module script">running a module script</a>
+            given |script|.
+
+        4. [=list/Append=] |promise| to |scripts|.
 
         Note: <a>Fetch a worklet script</a> won't actually perform a network request as it will hit
             the worklet's <a>module responses map</a>. It also won't have a parsing error as at this
             point it should have successfully been parsed by another worklet global scope. I.e.
             |script| should never be null here.
 
-    6. Run the <a>responsible event loop</a> specified by |insideSettings|.
+    9. [=Wait for all=] |scripts|, with success steps and failure steps both given by:
+
+        1. Run the <a>responsible event loop</a> specified by |insideSettings|.
```

Comment:
I think this isn't right. Without an inner event loop, the microtask queue (and thus any awaits) cannot run.

I see two possible semantics here:

- Ignore the promises returned; their errors will be reported appropriately, and that's all one would do with them.
- Run the module scripts "sequentially", waiting for one to complete (i.e. the promise to settle) before moving on to the next.

I think the first of these is probably more correct, since (IIRC) it aligns with what happens if you have multiple TLA-using `<script type=module>`s.